### PR TITLE
Delete performance/longrunning tests

### PR DESCRIPTION
**Pull request by Rob-Hague**
These do not seem very useful and are annoying to have to filter out when running tests via VS

### CODE REVIEW

File: src/Renci.SshNet.Tests/Classes/Common/ASCIIEncodingTest.cs
Comment (Rob-Hague) [inline src/Renci.SshNet.Tests/Classes/Common/ASCIIEncodingTest.cs:1]: Deleted this whole file: the custom ASCII class no longer exists, so it is just testing `System.Text.Encoding.ASCII`

**Comment by drieseng** [comment]
@Rob-Hague, would you consider transforming these tests into benchmarks (in a new project)?

**Comment by Rob-Hague** [comment]
Yes that would be good to have. I can work on it (at some point). Although I would still suggest that these particular benchmarks/tests do not add much value

**Comment by WojciechNagorski** [comment]
LGTM! Only one question. Did you add benchmarks for all these things in #1151

**Comment by Rob-Hague** [comment]
I did not, but I will

**Review by WojciechNagorski — APPROVED** [review]

**Pull request merged**